benchmark/README.md

```diff
@@ -0,0 +1,37 @@
+# Keras Benchmarks
+
+## Overview
+The benchmakr module aims to provide a performance comparison on different Keras backends using various models and dataset.
+Currently supported backends: TensorFlow, MXNet 
+
+## Setup
+To switch between different backends refer to this documentation:
+
+We provide benchmark scripts to run on ImageNet DataSet and Synthetic Dataset(randomly generated) 
+### ImnageNet Dataset
+There are two training mode: train_on_batch and fit_generator, we recommend train_on_batch since it's more efficient on multi_gpu.
```

Comment:
This is applicable for datasets that cannot fit into memory right? Please clarify it.
Put a link to Github discussion on keras-team/keras regarding these modes.
train_on_batch -> `train_on_batch`
fit_generator -> `fit_generator`